portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,35 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
+    <div id="content-container">
       <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <image src="images/pic.jpeg"/>
+      <h2>Introduction</h2>
+      <p>Hello, I am Deus. I am a rising junior at Princeton University, majoring in Computer Science.</p>
+      <h2>Background</h2>
+      <p>I was born in Kigali, Rwanda, in a family of two children, and I am the eldest.</p>
+      <p>Click here to see random pictures of me</p>
+      <button onclick="addRandomGreeting()">images</button>
+      <div id="image-container">
+	<img src="images/IMG-1944.jpg" alt="Image" height="170" width="170"/>
+      </div>
+      <h2>Experience</h2>
+      <ul>
+	<li>Google, STEP Intern, Summer 2020</li>
```

Comment:
When creating lists/blocks/tables make sure to indent child elements so it is east to tell how things are nested. 

E.g:

```
<ul>
  <li>foo</li>
  <li>bar</li>
</ul>

``` 
See: https://google.github.io/styleguide/htmlcssguide.html#General_Formatting

Reply:
Nvm I see these have already been fixed in the followup commit. :)